provider/filestore.go

```diff
@@ -289,7 +294,12 @@ func NewGCSFileStore(config Config) (FileStore, error) {
 		return nil, fmt.Errorf("could not deserialize config: %v", err)
 	}
 
-	creds, err := google.CredentialsFromJSON(context.TODO(), GCSConfig.Credentials.SerializedFile, "https://www.googleapis.com/auth/cloud-platform")
+	serializedFile, err := json.Marshal(GCSConfig.Credentials.SerializedFile)
```

Comment:
Should rename one of these. Unclear how serializing a serialized file returns a serialized file

Reply:
I will rename SerializedFile to `JSON`.

Reply:
that works